### ISSUE TITLE
Question about kernal-native L4 mode

### ISSUE THREAD

**Issue by Okabe-Rintarou-0** [kmesh-net/kmesh#998]
**Please provide an in-depth description of the question you have**:
In kernal-native L4 mode, we use cgroup/connect4, it can hook the tcp connection. But what hook should we use for tcp close? Why don't we use sockops like L7 mode? 
**What do you think about this question?**:

**Environment**:
- Kmesh version:
- Others:

**Comment by Okabe-Rintarou-0** [comment]
cc @hzxuzhonghu  @lec-bit

**Comment by hzxuzhonghu** [comment]
what do you want to do when tcp close? We have sockops hookpoint in bpf/kmesh/ads/sockops.c

**Comment by Okabe-Rintarou-0** [comment]
https://github.com/kmesh-net/kmesh/blob/50dafa8787c4b1f517f245016b44e31ecf191630/bpf/kmesh/bpf2go/bpf2go.go#L20C1-L25C1

we don't use sockops in non-enhanced mode, right? I need to maintain the connection number of a cluster

**Comment by hzxuzhonghu** [comment]
Yeah, it does not use sockops in non-enhanced mode. I doubt there is no fake connect in non enhanced mode. So how could itgoes into `cluster_manager`.  

maybe some others can shed light on it

**Comment by hzxuzhonghu** [comment]
@Okabe-Rintarou-0 I have consulted @supercharge-xsy , we believe you can turn on sockops for non-enhanced kernel, and keep in mind you also need to update the bpf loader.

**Comment by hzxuzhonghu** [comment]
cgroup/connect is useful
